rules/integrations/cloud_defend/execution_netcat_listener_established_inside_a_container.toml

```diff
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2023/04/26"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/05/08"
+
+[rule]
+author = ["Elastic"]
+description = "This rule detects an established netcat listener running inside a container. Netcat is a utility used for reading and writing data across network connections, and it can be used for malicious purposes such as establishing a backdoor for persistence or exfiltrating data."
+false_positives = ["""
+  There is a potential for false positives if the container is used for legitimate tasks that require the use of netcat, such as network troubleshooting, testing or system monitoring. It is important to investigate any alerts generated by this rule to determine if they are indicative of malicious activity or part of legitimate container activity.
+  """]
+from = "now-6m"
+index = ["logs-cloud_defend*"]
+interval = "5m"
+language = "eql"
+license = "Elastic License v2"
+name = "Netcat Listener Established Inside A Container"
+risk_score = 73
+rule_id = "a52a9439-d52c-401c-be37-2785235c6547"
+severity = "high"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution", "Container"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = """
+process where container.id: "*" and event.type== "start" 
+and event.action in ("fork", "exec") and not event.action== "end" and 
```

Comment:
I believe this line could be made simpler by just changing it to. This would capture any event action that's not end.
```sql
event.action != "end"